app/templates/components/nav-bar.hbs

```diff
@@ -46,7 +46,7 @@
       {{/if}}
       {{#if (not-eq routing.currentRouteName 'create')}}
         <div class="item">
```

Comment:
sir i have not made any changes to package-lock.json but its showing 901 deletion. please help me.
thank you